osbuild/objectstore.py

```diff
@@ -390,9 +389,5 @@ def cleanup(self):
         for obj in self._objs:
             obj.cleanup()
 
-    def __enter__(self):
-        return self
-
     def __exit__(self, exc_type, exc_val, exc_tb):
         self.cleanup()
-        return exc_type is None
```

Comment:
You say in the commit message:

> This commit cleans this up and just follows what the contextlib module does and **returns None explicitly everywhere.**

But here (and in the following lines) it is just removed, no?